.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     working_directory: ~/stackstorm-ha
     docker:
       # Pin Helm to v2.x, see https://github.com/StackStorm/stackstorm-ha/issues/98
-      - image: lachlanevenson/k8s-helm:v2.16.9
+      - image: lachlanevenson/k8s-helm:v3.3.4
```

Comment:
Following the Helm version change for Linting CircleCI job, it fails now here https://app.circleci.com/pipelines/github/StackStorm/stackstorm-ha/825/workflows/ec7af379-09b4-451e-a4a8-5ba367a82292/jobs/2991

with the error message:
```
+ helm init --client-only
Error: unknown command "init" for "helm"
```

We'll need to update the steps below as `helm init` command is not relevant anymore for the new Helm `v3`.

Reply:
removed the init as according to the Helm 2 to 3 upgrade notes this isn't required anymore.